install/kubernetes/quick-install.yaml

```diff
@@ -382,7 +382,7 @@ spec:
               name: cilium-config
               optional: true
         image: "docker.io/cilium/cilium:latest"
```

Comment:
These image tags should be updated too, no?

Reply:
Fixed!